packages/devtools/cli/src/commands/reset/index.ts

```diff
@@ -57,7 +57,15 @@ export default class Reset extends BaseCommand<typeof Reset> {
         if (await daemon.isRunning(this.flags.profile)) {
           await daemon.stop(this.flags.profile, { force: this.flags.force });
         }
-      });
+      }, false);
+      // TODO(egorgripasov): Better compatibility with Phoenix daemon.
+      try {
+        await this.execWithDaemon(async (daemon) => {
+          if (await daemon.isRunning(this.flags.profile)) {
+            await daemon.stop(this.flags.profile, { force: this.flags.force });
+          }
+        }, true);
+      } catch (err) {}
```

Comment:
Document silent error. or `// Ignore.` if expected.

Reply:
fixed in further PRs